lib/send_sms.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+
+module SMS
+
+  def send_confirmation_sms number, message
+    Dotenv.load
+    account_sid = ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+    client = Twilio::REST::Client.new account_sid, auth_token
+    twilio_no = "+441548312031"
+
+    client.account.messages.create(
+      :from => twilio_no,
+      :to => number,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)